scripts/deploy/deploy-webapi.ps1

```diff
@@ -27,12 +27,12 @@ param(
     $PackageFilePath = "$PSScriptRoot/out/webapi.zip",
 
     [switch]
-    # Switch to add our URIs in app registration's redirect URIs if missing
-    $EnsureUriInAppRegistration,
+    # Don't attempt to add our URIs in app registration's redirect URIs
```

Comment:
Yes, this changes the existing logic.  But given this the current logic has only worked from this week and that people are already complaining about how it is incomplete, I'm fixing it now to kill this problem right away.

Reply:
nit: maybe worth mentioning it's the WebApp's or the frontend's end registration.

Reply:
Done

Reply:
Done